lib/public_key/test/public_key_SUITE.erl

```diff
@@ -1132,8 +1132,11 @@ pkix_verify_hostname_options(Config) ->
     true = public_key:pkix_verify_hostname(Cert, [{dns_id,"abb.bar.example.com"}]),
     false = public_key:pkix_verify_hostname(Cert, [{dns_id,"example.com"},
                                                    {dns_id,"abb.bar.example.com"}],
-                                            [{fqdn_fun,fun(_)->undefined end}]).
-    
+                                            [{fqdn_fun,fun(_)->undefined end}]),
+    {ok, PrefixBin} = file:read_file(filename:join(DataDir,"prefix-dots.pem")),
+    PrefixCert = public_key:pkix_decode_cert(element(2,hd(public_key:pem_decode(PrefixBin))), otp),
+    true = public_key:pkix_verify_hostname(PrefixCert, [{dns_id,"..a"}]),
```

Comment:
is `public_key:pkix_verify_hostname(PrefixCert, [{dns_id,"a"}])` interesting to test as well?
it appeared in the discussion AFAIK ...

Reply:
No the problem was with more than one dot.